cmd/getroughtime/main.go

```diff
@@ -75,9 +76,15 @@ func main() {
 		} else if len(pk) != 32 {
 			logger.Fatalf("Public key decode error: incorrect length")
 		}
+		switch *pingVersion {
+		case "", "Google-Roughtime", "IETF-Roughtime":
```

Comment:
If you take my suggestion above, take this one too:
```suggestion
		case "Google-Roughtime", "IETF-Roughtime":
```